### ISSUE TITLE
File extensions

### ISSUE THREAD

**Issue by HugoNikanor** [HugoNikanor/computerCraftUtils#5]
All the lua files should probably be changed to have .lua extensions.
Note that this will require updating the installer, as well as the apiloader
(and maybe something more)

This is since it's kinda silly that the program is currently noted as a python
project.

**Comment by Math-ias** [comment]
Yay! A use for the script I made! (Even though it causes the current problem.)

**Comment by Math-ias** [comment]
Would the `.lua` extension look a little weird on the computercraft computers? Should the installer install them with no extension or with the .lua extension?

EDIT: Your issue seems to suggest that we wish to keep the .lua, got it! 
EDIT2: Will os.loadApi be a problem? I don't want to have to access apis like _G["example.lua"] to use functions? It might be we worth it to, in the installer, remove the .lua extension.

**Comment by Math-ias** [comment]
I haven't tested my code yet. @HugoNikanor Did you? Before you merge into master and all ...

**Comment by HugoNikanor** [comment]
I'm in the process of testing it right now.

There where some small problems (lua being one-indexed created a problem with the substrings)

Once this is done I'm going to write better instructions for where to merge new commits.

**Comment by HugoNikanor** [comment]
Issue should now be resoldev
